pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -218,6 +224,31 @@ func extractLibInfo(pod *corev1.Pod, containerRegistry string) []libInfo {
 	return libInfoList
 }
 
+// extractLibrariesFromOwnerAnnotations constructs the libraries to be injected if the languages
+// were stored in workloadmeta store based on owner annotations.
+func extractLibrariesFromOwnerAnnotations(pod *corev1.Pod, registry string) []libInfo {
```

Comment:
I think we can rename it as `extractLibrariesFromLangDetectionAnnotations` as `OwnerAnnotations` is not clear enough

Reply:
I chose this naming because we have another function `extractLibrariesFromAnnotations` ([here](https://github.com/DataDog/datadog-agent/blob/33b68ba92afcb68a7c339b95774688d65fd2bd76/pkg/clusteragent/admission/mutate/auto_instrumentation.go#L221)) which calculates the libraries to inject based on the language annotations on the pod itself. 

Having another function with the name `extractLibrariesFromLangDetectionAnnotations` would sound confusing to me.

What about `extractLibrariesFromAutoLangDetectionAnnotations` ?

Reply:
Why not. `OwnerAnnotations` might sound too ambiguous for someone without context.

Reply:
My 2 cents, the current name is ok (and it is a private function).

We can add more context in the function description.

Reply:
Tbh as a maintainer without the context I would have no idea of what `OwnerAnnotations` are

Reply:
I have improved the description comment to clarify what an owner is (As suggested by @clamoriniere)